ingest/ingest_pipeline.py

```diff
@@ -483,22 +483,35 @@ def subsample(self):
         return 0
 
     @custom_metric(config.get_metric_properties)
-    def ingest_anndata(self):
-        """Ingests anndata files."""
+    def extract_from_anndata(self):
+        """Extract data subsets from anndata file per SCP filetypes."""
         self.anndata = AnnDataIngestor(
             self.anndata_file, self.study_id, self.study_file_id, **self.kwargs
         )
         if self.anndata.validate():
             self.report_validation("success")
-            if self.kwargs["extract_cluster"] == True:
+            study_info = config.get_metric_properties()
+            accession = study_info.get_properties()['studyAccession']
+            file_id = study_info.get_properties()['fileName']
+            outfile_prefix = f"{file_id}.{accession}"
```

Comment:
Do we really need the study accession here?  It seems redundant...  It would make the above method to set output filenames more regular if we don't have to account for it here.

Reply:
Agreed. Modified structure and naming of extracted files (see updated PR description above). `86ebd57`